src/openms/include/OpenMS/ANALYSIS/OPENSWATH/TransitionTSVFile.h

```diff
@@ -278,6 +305,10 @@ namespace OpenMS
     /// Populate a new TargetedExperiment::Protein object from a row in the csv
     void createProtein_(String protein_name, const String& uniprot_id,
                         OpenMS::TargetedExperiment::Protein& protein);
+    
+    /// Populate a new TargetedExperiment::Oligo object from a row in the csv
+    void createOligo_(String oligo_name,
```

Comment:
const& ?

Reply:
it's not const in `createProtein_`. Should it be in both?

Reply:
I did not check the implementation but would assume it doesn't change the protein_name or oligo_name so this might be an oversight